user/probe_gnutls.go

```diff
@@ -3,7 +3,7 @@ package user
 import (
 	"bytes"
 	"context"
-	"ecapture/assets"
+	assets "ecapture/assets"
```

Comment:
unnecessary.

Reply:
copy that, just because My ide reminds me that may be a mistake.